### DISCUSSION TITLE
Segregation of logs

### DISCUSSION THREAD

**Discussion by jawwad1995** [wazuh/wazuh#18063] [Q&A]
Hello,

We have a lot of applications each of them has its separate databases which generates its corresponding logs. Currently I am using IBM Q-radar and the logs of databases and application logs are merged. I want to separate these logs with the help of Wazuh. 
Is Wazuh able to segregate logs and meet my requirements.

**Comment by mdiego92** [comment]
Hello @jawwad1995, 

Could you please share with me a few log samples and verify which rules are triggered to assist you? It would be really helpful if you can also point what do you need to separate from them.

Looking forward to your response!

Best regards.
